src/catalog/query_metrics_catalog.cpp

```diff
@@ -26,7 +26,22 @@ QueryMetricsCatalog *QueryMetricsCatalog::GetInstance(
 }
 
 QueryMetricsCatalog::QueryMetricsCatalog(concurrency::Transaction *txn)
-    : AbstractCatalog(QUERY_METRICS_CATALOG_NAME, InitializeSchema().release(),
+    : AbstractCatalog(QUERY_METRICS_CATALOG_NAME,
```

Comment:
Actually, parameter QUERY_METRICS_CATALOG_NAME is passed twice and can be eliminated...